src/main.cpp

```diff
@@ -57,9 +55,9 @@ MacroMap()
     std::ifstream biome_file = files::open_data_file("biome_data.json");
     biome_file >> biome_data;
 
-    terrain::Terrain::generate_macro_map(64, 64, biome_data["Biome_1"]["Terrain_Data"]);
+    terrain::TerrainBase::generate_macro_map(64, 64, biome_data["Biome_1"]["Terrain_Data"]);
```

Comment:
Need a better name for `TerrainBase`

Also no magic numbers

Reply:
any sudgestions?

Reply:
Oh wait sorry nevermind about the name.
But get rid of the magic numbers